### ISSUE TITLE
Bug in argmax with `dj.U`

### ISSUE THREAD

**Issue by fabiansinz** [datajoint/datajoint-python#270]
Consider the following schema (added in #269).
```python
@schema
class ArgmaxTest(dj.Lookup):
    definition = """
    primary_key     : int
    ---
    secondary_key   : char(2)
    val             : float
    """

    n = 10

    @property
    def contents(self):
        n = self.n
        yield from zip(range(n ** 2),
                       itertools.chain(*itertools.repeat(
                                         tuple(map(chr, range(100, 100 + n))), n)),
                       np.random.rand(n ** 2))
```

If I want to perform an argmax for each secondary key, the syntax should be 
```python
(dj.U('val') * rel) & dj.U('secondary_key').aggr(rel, val='min(val)')
```
However, the only syntax that works is
```python
(dj.U('val') * rel) * dj.U('secondary_key').aggr(rel, val='min(val)')
```

Shouldn't they both give the same result?

**Comment by dimitri-yatsenko** [comment]
yes, seems right.

**Comment by eywalker** [comment]
This should be fixed with #271 and appropriate tests were added. @fabiansinz please verify the fix and I'll close this issue.
